server/service/src/invoice/inventory_adjustment/add_new_stock_line/generate.rs

```diff
@@ -78,6 +78,7 @@ pub fn generate(
         requisition_id: None,
         clinician_link_id: None,
         backdated_datetime: None,
+        diagnosis_id: None,
```

Comment:
There's a tonne of these, manually added to each one, as I think in general we probably want to think about each use case when adding something to invoice, however `..Default:default()` seems rather tempting at this stage?
@andreievg